app/main.py

```
@@ -1,122 +1,21 @@
-from pymongo import MongoClient
-import os
-from fastapi.middleware.cors import CORSMiddleware
 from bson import json_util
 from bson.objectid import ObjectId
 from typing import Annotated
 import uuid
-from prometheus_client import Counter, Histogram, Gauge, make_asgi_app, CollectorRegistry
-from starlette.requests import Request
 from argon2 import PasswordHasher
 from argon2.exceptions import VerifyMismatchError
+import mariadb
```

Ruff (0.12.2)

`mariadb` imported but unused

Remove unused import: `mariadb`

(F401)